Paper.tex

```diff
@@ -1156,7 +1156,7 @@ \section{Acknowledgements}
 
 \section{Availability}
 
-The source of this paper is maintained at \url{https://github.com/ethereum/yellowpaper/}. An auto-generated PDF is located at \url{https://ethereum.github.io/yellowpaper/paper.pdf}.
+The source of this paper is maintained at \url{https://github.com/ethereum/yellowpaper/} and is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}. An auto-generated PDF is located at \url{https://ethereum.github.io/yellowpaper/paper.pdf}.
```

Comment:
If you change it to Creative Commons Attribution **Share-Alike** 4.0 this will be in line with #402 and can be merged easily.

Reply:
Right, yes, I can do that.  https://creativecommons.org/faq/#can-i-combine-material-under-different-creative-commons-licenses-in-my-work